config/alerts.yaml

```diff
@@ -0,0 +1,90 @@
+# etcd
```

Comment:
We need to update the docs here: https://github.com/redhat-chaos/krkn/blob/main/docs/metrics.md#scraping-and-storing-metrics-for-the-run.